src/pages/Filing/FilingApp/FilingWarnings.tsx

```diff
@@ -0,0 +1,197 @@
+import submitWarningsAccept from 'api/requests/submitWarningsVerified';
+import useSblAuth from 'api/useSblAuth';
+import classnames from 'classnames';
+import FormWrapper from 'components/FormWrapper';
+import {
+  Alert,
+  Button,
+  Checkbox,
+  Heading,
+  Icon,
+  List,
+  ListLink,
+  TextIntroduction,
+  WellContainer,
+} from 'design-system-react';
+import { useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+import type { SubmissionResponse } from 'types/filingTypes';
+import { FileSubmissionState } from 'types/filingTypes';
+import useGetSubmissionLatest from 'utils/useGetSubmissionLatest';
+import useInstitutionDetails from 'utils/useInstitutionDetails';
+import { NavigationPrevious } from './FilingNavButtons';
+import { FilingSteps } from './FilingSteps';
+import InstitutionHeading from './InstitutionHeading';
+
+const isSubmissionAccepted = (submission?: SubmissionResponse): boolean => {
+  return submission?.state === FileSubmissionState.SUBMISSION_ACCEPTED;
+};
+
+function FilingWarnings(): JSX.Element {
+  const navigate = useNavigate();
+  const auth = useSblAuth();
+  const { lei, year } = useParams();
+  const [boxChecked, setBoxChecked] = useState(false);
+  const [formSubmitLoading, setFormSubmitLoading] = useState(false);
```

Comment:
Added a TODO for refactoring to use useMutate but I don't want to spend time right now refactoring and testing since we have an approach that works, and a lot of other tasks that would be a better use of time/effort.

Reply:
Sounds good, can we get a ticket for that?

Reply:
@billhimmelsbach https://github.com/cfpb/sbl-frontend/issues/463